pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[tool.poetry]
+name = "pelican-read-more"
+version = "0.0.0"
+description = "Pelican plugin that adds an inline “Read More…” link"
+authors = ["Pelican Dev Team <authors@getpelican.com>"]
+license = "AGPL-3.0"
+readme = "README.md"
+keywords = ["pelican", "plugin"]
```

Comment:
Might be useful to add further keywords? e.g.

`keywords = ["pelican", "plugin", "article", "summary", "readmore"]`

Reply:
Sure, why not. Thanks!